config/master/aws-k8s-cni-cn.yaml

```diff
@@ -121,7 +122,7 @@
             "command":
             - "/app/grpc-health-probe"
             - "-addr=:50051"
-          "initialDelaySeconds": 35
+          "initialDelaySeconds": 10
```

Comment:
Unless the CNI has a lot of pods and ENIs attached, the pod will get ready in just a few seconds. Having the initial delay be 35 seconds is not needed for the readiness probe.

Reply:
Why not zero (no initialDelay) ?

Reply:
True, we might as well set it to 0. 😄

Reply:
Hmm.. Maybe 1? I've never seen it start faster than that.. 😀